examples/word-count/.template/pre-script.rhai

```diff
@@ -1,3 +1,4 @@
 variable::set("PYO3_VERSION", "0.20.0");
 file::rename(".template/Cargo.toml", "Cargo.toml");
+file::rename(".template/pyproject.toml", "pyproject.toml");
```

Comment:
This was missing from the template, thought I would fix at the same time while touching up these examples.